src/main/java/seedu/mtracker/console/InputParser.java

```diff
@@ -73,6 +75,15 @@ public DeleteCommand getDeleteInstrumentCommand(String[] commandComponents, Arra
         return deleteCommand;
     }
 
+    public DoneCommand getDoneInstrumentCommand(String[] commandComponents, ArrayList<Instrument> instruments)
+            throws InvalidIndexError, InvalidNoIndexError, InvalidBoundsError {
+        DoneCommand doneCommand = new DoneCommand();
+        getIndexNumber(commandComponents);
+        validateIndexWithinBounds(instruments);

```

Comment:
Can help abstract this out to another function, I realise its going to be used one more time with edit also